src/PKSim.Core/Mappers/IndividualToIndividualBuildingBlockMapper.cs

```diff
@@ -149,7 +149,7 @@ private void addOriginDataToBuildingBlock(IndividualBuildingBlock buildingBlock,
       private string originDataFormattedForDisplay(OriginDataParameter parameter)
       {
          var dimension = _dimensionRepository.DimensionForUnit(parameter.Unit);
-         var unit = dimension.FindUnit(parameter.Unit);
+         var unit = dimension.UnitOrDefault(parameter.Unit);
```

Comment:
FindUnit return null if not found. Not good

Reply:
I don't understand the problem.
How can it happen that `dimension.FindUnit(parameter.Unit)` returns null, when the dimension was retireved for precisely this unit via `_dimensionRepository.DimensionForUnit(parameter.Unit)`?

Reply:
Unit empty is not saved to the snapshot presumably. It’s in fact null when looking at the loaded object. Maybe the find unit could do something special for the case of dimensionsless. But right now it doesn’t support this.